### PR TITLE
Creation layout

### PR DESCRIPTION
**Pull request by biafra23**
Handles #936 

**I think we need to make a decision on whether the last separator should be part of a custom view. And then make this consistent throughout the app.**

Changes proposed in this pull request:
- Remove additional separator by extracting it from the mastercopy custom view
- Changing color of item separator in tx list (according to the zeplin layouts)
- Fixing height of several separators
- Fixing bottom margin for tx hash
- Fixing bottom margin for created date
- Fixing separator color and height for detail views
- Added ids to several separators to make layout debugging easier
- Added custom attribut to NamedAddressItem to disable/enable separator

**Notes for QA**: This might break the layout of tx list, tx details,  app settings, collectible details and safe settings. 

Design: https://zpl.io/amAw7dK

Before | After 
------- | -------
Screenshot_20201028-134751 | Screenshot_20201028-134541

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/res/layout/fragment_settings_safe.xml [hunk 1]
```diff
@@ -106,6 +106,13 @@
                     android:background="@drawable/background_selectable_white"
                     app:setting_openable="false" />
 
+                <View
+                    android:id="@+id/master_copy_separator"
+                    android:layout_width="match_parent"
+                    android:layout_height="2dp"
```
Comment (jpalvarezl) [inline app/src/main/res/layout/fragment_settings_safe.xml:112]: Is this double size? Should we use `@dimen/item_separator_height`?

**Review by jpalvarezl — APPROVED** [review]
The after screenshot looks great :D Really happy to see stuff parametrised!

File: app/src/main/res/layout/fragment_settings_safe.xml (hunk 1 above)
Comment (biafra23) [reply app/src/main/res/layout/fragment_settings_safe.xml:112]: I missed one. This should also be `@dimen/item_separator_height` which is 2dp

Comment (biafra23) [reply app/src/main/res/layout/fragment_settings_safe.xml]: Fixed in dfe45c2bdd12532d7f1091d8eb32a29fde1ef724

**Review by elgatovital — APPROVED** [review]

**Pull request merged**